tests/unit/handlers/ingestionHandler.spec.ts

```diff
@@ -699,4 +791,71 @@ describe('IngestionJobHandler', () => {
       });
     });
   });
+
+  describe('callback handling', () => {
```

Comment:
by the design we also should throw an callback on failed process - should we should add test for it

Reply:
will be implemented on next PR